src/keypair.rs

```diff
@@ -62,6 +66,9 @@ impl ReadWrite for Keypair {
                 writer.write_all(&key.to_vec())?;
                 writer.write_all(&key.public_key.to_vec())?;
             }
+            helium_crypto::Keypair::Secp256k1(_) => {
+                return Err(anyhow!("Secp256k1 key type for write."))
```

Comment:
I think the anyhow macro for this pattern is `bail!`

Reply:
Ah nice - I had forgotten about that one.